core/services/assets/Registry.php

```diff
@@ -176,7 +177,10 @@ public function scripts()
                 true
             );
             $this->jsdata['eejs_api_nonce'] = wp_create_nonce('wp_rest');
-            $this->jsdata['paths'] = array('rest_route' => rest_url('ee/v4.8.36/'));
+            $this->jsdata['paths'] = array(
+                'rest_route' => rest_url('ee/v4.8.36/'),
+                'collection_endpoints' => EED_Core_Rest_Api::getCollectionRoutesIndexedByModelName()
+            );
```

Comment:
This is provided so there's an easy way for any of our React work to receive a list of collection endpoints for setting up EE REST API queries.

Reply:
Hmm I don't think it's a super expensive request.  The unfortunate thing is I don't think your suggestion would work because this is actually getting generated on a _normal_ browser request *not* an api request.  It's to provide info to the js for _making_ api requests.

Reply:
ahhh kk  then nvmd